lib/ash/query/aggregate.ex

```diff
@@ -230,6 +239,19 @@ defmodule Ash.Query.Aggregate do
           build_opts -> build_query(target_resource, resource, build_opts)
         end
 
+      # Set bypass context IMMEDIATELY if multitenancy bypass is requested
+      query =
+        if opts.multitenancy == :bypass do
+          query
+          |> Ash.Query.set_tenant(nil)
```

Comment:
There is a separate discussion about not actually clearing the tenant in these scenarios, but instead just skipping actually using the tenant. That way you can see what tenant was bypassed and what wasn't. Lets try that here.

Reply:
@zachdaniel I deleted the `Ash.Query.set_tenant(nil)`, we do not need it, because i implemented  here to know what aggregations are bypass what are not

https://github.com/ash-project/ash/pull/2427/files#diff-43bd00d31554df0f454a25914f2291b8e66933c876900039fecfaf5d0e3345e2R91-R143

I just wanted to follow your codebase, but we do not need it here 🙏🏻